### ISSUE TITLE
For Indian Number Plate I am getting blank space as output

### ISSUE THREAD

**Issue by tallamtharunsai** [openvinotoolkit/training_extensions#405]
I have used the plain data like black and white image for training the model. 
The training loss is also very high. 15
![image](https://user-images.githubusercontent.com/64833322/103399008-1ae61b00-4b65-11eb-8031-40b708467312.png)
and i have used the pattern for the eval, that is : '^[A-Z]{2} [0-9]{2} [A-Z]{2} [0-9]{4}$' for this.
I am getting the blank spaces as output
![new_dataset_out](https://user-images.githubusercontent.com/64833322/103399085-66002e00-4b65-11eb-9a1f-02b645f58649.JPG)
Then i have aging checked if the whole thing is working or not. So, i have tried for pretrained Chinese data and the outputs of it are like this.
![chinese_out](https://user-images.githubusercontent.com/64833322/103399187-c55e3e00-4b65-11eb-9e42-51528ad0e6ba.JPG)
![chinese_out3](https://user-images.githubusercontent.com/64833322/103399197-ce4f0f80-4b65-11eb-9f74-ffe25107e65c.JPG)

the packages are uptodate

**Comment by tallamtharunsai** [comment]
My bad. I have done it with different batch size but kept the learning_rate and grad_noise_scale same. So, the model is not converged.
